src/random-seed/random-seed.c

```diff
@@ -106,9 +107,11 @@ static int run(int argc, char *argv[]) {
         _cleanup_close_ int seed_fd = -1, random_fd = -1;
         bool read_seed_file, write_seed_file, synchronous;
         _cleanup_free_ void* buf = NULL;
+        struct sha256_ctx hash_state;
+        uint8_t hash[32];
```

Comment:
move to innermost scope?

Reply:
`hash_state` is already innermost, because it gets updated in two scopes later. But `hash` could be moved to an innerscope.